### DISCUSSION TITLE
It says type code . to open all files in directory. Shouldnt it be code *?

### DISCUSSION THREAD

**Discussion by qasimala** [TheOdinProject/curriculum#23657] [Q&A] [answered]
In the installations section of the foundations chapter, it says to type "code ." to open all files in that directory. However, on every terminal that I've tried, that doesn't work. A new empty vscode window just pops up. I was wondering if this is for a specific terminal or is this a mistake? 

What works for me on every terminal is "code *". If this is a mistake, I can do a pull request and make the change but I feel like I'm missing something. That's why I decided to ask here. 
Thanks.

**Comment by xandora** [comment, marked as answer]
Good catch. It should say something like:

> [...] if you have VS Code installed, you can `cd` into the project directory and then type `code .` (with the period) to open up that folder in VS Code.

**Reply by isaac-Makism** [reply]
If it seems to you a little tricky to open your vscode from the command line, try a different way to open the vscode by the graphical interface just click on the icon then open the command line on your vscode cd into the directory and code . (always with the period) to easily open the folder intended in your IDE.
